ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -227,13 +238,26 @@ - (void)webView:(__unused WKWebView *)webView decidePolicyForNavigationAction:(W
   NSString* scheme = url.scheme;
   
   BOOL isJSNavigation = [scheme isEqualToString:RCTJSNavigationScheme];
-  
+
+  static NSDictionary<NSNumber *, NSString *> *navigationTypes;
```

Comment:
What's the benefit of returning navigationTypes as string? Since it's hardcoded anyway, I think it would be possible to put the logic in JS side? Also this will break compatibility so we might have to bump a major version again.